http.go

```diff
@@ -186,56 +187,65 @@ func (i *responseInterceptor) Header() http.Header {
 // TracedHTTPHandler returns a http.Handler that is traced as an opentracing.Span
 func TracedHTTPHandler(
 	h http.Handler,
+	ignoredPaths *regexp.Regexp,
```

Comment:
actually we should use the interceptor for this.  you can add an ignore field to the SpanConfig and set that however you want in the interceptor.